Doppler.ContactPolicies.Business.Logic/Class1.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Doppler.Contact.Policies.Business.Logic
+namespace Doppler.ContactPolicies.Business.Logic
```

Comment:
What do you think about using this PR to rename the business logic layer too:

```suggestion
namespace Doppler.ContactPolicies.BusinessLogic
```

Reply:
I will in another PR, i dont want to mess with filenames and so on